doc/manpages/regex.asciidoc

```diff
@@ -161,11 +161,11 @@ preceded by `bar` and where `foo` matches from the current position
 Modifiers
 ---------
 
-Some modifiers can control the matching behaviour of the atoms following
+Some modifiers can control the matching behavior of the atoms following
```

Comment:
What ! American spelling ? This goes too far mister.

Reply:
The rest of the documentation is in american english as well, so I stayed within the same lexicon, even though I personally prefer the queen's english.